com.microsoft.mrtk.input/Subsystems/SyntheticHands/SyntheticHandsSubsystem.cs

```diff
@@ -52,28 +53,89 @@ private static void Register()
         /// <returns>
         /// Identifier representing the hand pose.
         /// </returns>
-        public GestureId GetNeutralPose(XRNode handNode)
-        {
-            return (provider as SynthesisProvider).GetNeutralPose(handNode);
-        }
-
+        [Obsolete("Please use the GetNeutralGesture(handNode) instead.")]
+        public GestureId GetNeutralPose(XRNode handNode) => GetNeutralGesture(handNode);
         /// <summary>
         /// Sets the neutral pose for the specified hand.
         /// </summary>
         /// <param name="handNode">The hand for which the neutral pose is being set.</param>
         /// <param name="poseId">The desired hand pose.</param>
-        public void SetNeutralPose(XRNode handNode, GestureId poseId)
+        [Obsolete("Please use the SetNeutralGesture(handNode, poseId) instead.")]
+        public void SetNeutralPose(XRNode handNode, GestureId poseId) => SetNeutralGesture(handNode, poseId);
```

Comment:
I almost wonder if we should move the other direction. Instead of Pose -> Gesture, if we should align on the Pose terminology. To me, "gesture" implies some amount of motion, like a change from one pose to another. Whereas "pose" to me implies more of a specific hand position, which these entries and the `GestureId` (`PoseId`?) enum represent as well.

Reply:
I see where you're coming from on that front, but UnityEngine itself has a concept of a Pose as a struct with position + rotation https://docs.unity3d.com/ScriptReference/Pose.html. With that in mind, I'd much rather keep our gesture terminology, as we've been using that consistently since MRTK2. Things like "thumbs up" semantically aligns closer with the gesture terminology in my opinion as well.

Reply:
+1 on @RogPodge 's point. Also, other frameworks are using "Gesture" as a term for a particular arrangement/shape of a hand that can be recognized and trigger an action (thumbs up, pinch, peace, spock, etc). I'm good with Pose = (position,rotation,etc) and Gesture = (collection of joint poses).

Reply:
I think the Qualcomm stuff uses Gesture in this way. Meta Hand Interaction stuff does too, I think?

Reply:
Not to get too in the weeds, but I still think those descriptions are distinct from what this represents. A "pinch gesture" would be the act of pinching imo, whereas these values describe the two poses on either side of such a gesture (an open pose and a pinched pose, and moving from one to the other would be a pinch gesture).

Even the dictionary definition of "gesture" (I know, I know...) is "a movement of part of the body, especially a hand or the head, to express an idea or meaning.". I think gesture implies more action than this `GestureId` really represents.

If the sticking point is the word "pose", we can always find a synonym (posture, shape, idk)

Reply:
(I also don't think I care enough to die on this hill 😅 but I do think there's an important enough distinction for what this enum represents)

Reply:
I think it'd be better to reconsider any naming considerations in a future PR, since the refactoring considerations are getting a bit entangled with initial goal of adding some the simulator improvements.

Reply:
Due to discussions we had, we're actually moving to rename `Gestures` to `Handshapes` in this PR. The reason we are doing this is because "Gesture" has an implication of motion, which these enums do not really prescribe. Based on pre-existing sign language terminology, the word "Handshape" best captures the intended use case, which is a representation of a collection of HandJointPoses used to represent the shape that the hand forms.